tests/subsys/bluetooth/fast_pair/storage/account_key_storage/src/main.c

```diff
@@ -93,7 +93,7 @@ ZTEST(suite_fast_pair_storage_common, test_one_key)
 	struct fp_account_key account_key;
 
 	cu_generate_account_key(seed, &account_key);
-	err = fp_storage_ak_save(&account_key);
+	err = fp_storage_ak_save(&account_key, NULL);
```

Comment:
Consider adding e.g. a build assert to clearly indicate that AK storage tests do not support Fast Pair bond management (similar limitation may apply to the factory reset test)